facets_mcp/utils/ftf_command_utils.py

```
@@ -9,33 +9,25 @@
+def run_ftf_command(command: list) -> str:
     """
-    Runs an FTF command using the Click test runner.
-    
+    Runs an FTF command using subprocess to avoid dependency conflicts.
     Args:
         command (List[str]): The FTF command as a list of strings.
-        
     Returns:
         str: The output from the command execution.
     """
     if not command[0] == 'ftf':
```

Ruff (0.11.9)

Use `command[0] != 'ftf'` instead of `not command[0] == 'ftf'`

Replace with `!=` operator

(SIM201)